client/src/pages/logs-screen/LogTable.tsx

```diff
@@ -28,36 +31,40 @@ interface LogItemInf {
   logType: string;
   created_at: string;
   metaTimestamp?: string; // @Todo will rename it once added to backend
```

Comment:
No need of this line, Full string data come under `meta`

Reply:
Okaay done